### PR TITLE
[ADD] hr_payroll_run_report_xls

### PR DESCRIPTION
**Pull request by saltonmassally**
Download a report of payslip run slips as excel

### CODE REVIEW

File: hr_payroll_run_report_xls/wizard/print_run_report.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+				<form string="Print Run Report">
+					<footer>
+						<button name="print_report" string="Print Register" type="object"
+							context="{'xls_export':1}" default_focus="1" class="oe_highlight" />
+						or
+						<button string="Cancel" class="oe_link" special="cancel" />
+					</footer>
+				</form>
+			</field>
+		</record>
```
Comment (ghost) [inline hr_payroll_run_report_xls/wizard/print_run_report.xml]: can you put indentation to 4 spaces or 2 spaces please. This is a bit unreadable.

File: hr_payroll_run_report_xls/README.rst [hunk 2]
```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+========================
+Payslip Run Excel Report
+========================
+
+* Download a report of payslip run slips as excel
```
Comment (ghost) [inline hr_payroll_run_report_xls/README.rst:8]: can you add in markdown an example of what the report looks like

**Comment by saltonmassally** [comment]
@dufresnedavid how do I get past this "You try to install module 'hr_payroll_run_report_xls' that depends on module 'report_xls'.\nBut the latter module is not available in your system" thanks

**Comment by dreispt** [comment]
It's from the https://github.com/OCA/reporting-engine repo.
Edit `oca_dependencies` at the repo root and add to it the line:

```
reporting-engine
```

**Comment by saltonmassally** [comment]
Sorry I think I have botched this one, didn't have the oca-dependencies file so I pulled in the lastest changes... I am going to close this and reopen here https://github.com/OCA/hr/pull/164

**Comment by dreispt** [comment]
A new PR was not needed: you can add commits or even amend / rewrite these ones.
In fact, that's friendlier for reviewers than opening new PRs.

**Comment by saltonmassally** [comment]
Sorry about this and thanks for pointing it out; I am relatively new to this level of collaboration and team work but I am a fast learner

**Comment by dreispt** [comment]
No problem! Keep up with the good work.

**Pull request closed without merging**